src/Faker/Provider/fr_CH/Person.php

```diff
@@ -87,4 +87,87 @@ class Person extends \Faker\Provider\fr_FR\Person
         'Waeber', 'Weber', 'Wenger', 'Widmer', 'Wyss',
         'Zbinden', 'Zimmermann',
     );
+
+    /**
+     * Generates a valid random AVS13 (swiss social security) number
+     *
+     * This function will generate a valid random AVS13 number and return it
+     * as a formatted string.
+     *
+     * @see https://www.zas.admin.ch/zas/fr/home/partenaires-et-institutions-/navs13.html
+     * @return string
+     */
+    public function navs13()
+    {
+        $b = array(
+            756,
+            self::generateRandomDigits(4),
+            self::generateRandomDigits(4),
+            self::generateRandomDigits(1),
+        );
+
+        $checksum = self::computeAvs13Checksum(implode("", $b));
+
+        $avs = sprintf("%s.%s.%s.%s%s", $b[0], $b[1], $b[2], $b[3], $checksum);
+
+        return $avs;
+    }
+
+
+    /**
+     * Generates a string of {@see $length} random digits
+     *
+     * This function will generate a {@see $length} digits string. The length of
+     * the string must be greater than 0. If the provided length is lower or
+     * equal to 0, an \OutOfBoundsException is thrown.
+     *
+     * By default, if no {@see $length} parameter is given, the function will
+     * return a one-digit string.
+     *
+     * @since 0.1.0
+     * @param int $length Length of the string of digits to generate
+     * @return string A string of {@see $length} random digits
+     * @throws \OutOfBoundsException
+     */
+    protected static function generateRandomDigits($length = 1)
```

Comment:
I reckon this kind of method should go into `Faker/Provider/Base` but you probably could get the same result using `numberBetween`